tools/lsp/preview/ui.rs

```diff
@@ -32,8 +32,11 @@ pub fn create_ui(style: String) -> Result<PreviewUi, PlatformError> {
 
     ui.set_known_styles(style_model.into());
     ui.set_current_style(style.clone().into());
-    #[cfg(feature = "experimental")]
-    ui.set_experimental(true);
+    ui.set_experimental(
+        std::env::var_os("SLINT_PREVIEW_ENABLE_EXPERIMENTAL_FEATURES")
```

Comment:
I'd re-use the env variable we already have:

```suggestion
        std::env::var_os("SLINT_ENABLE_EXPERIMENTAL_FEATURES")
```

Yes i know, this will also enable support for the ComponentContainer, but i don't see the need to have them in different gates.